src/containers/MovieRatings/MovieRatings.js

```diff
@@ -15,40 +15,66 @@ export const MovieRatings = ({movieId, rating, user, allRatings}) => {
     }
 
     return await postRating(userRating, user)
-                .then(data => rating(data))
-}
+      .then(data => rating(data))
+  };
+
+  const handleDelete = (event) => {
+    event.preventDefault()
+    removeRating(user, getRatingId(movieId))
+    .then(data => {
+      fetchRatings(user).then(ratingData => {
+        deleteRating({ ratings: ratingData.ratings });
+    });
+  });
+};
+
+  const getRatingId = movieId => {
+    const movieIds = allRatings.map(rating => rating.movie_id);
+    if (movieIds.includes(movieId)) {
+      return allRatings.find(movie => movie.movie_id === movieId).id;
+    }
+  };
+
   const  handleClick = async (event) => {
     const rate = parseInt(event.target.id)
     return await setRating(rate)
-  }
-
-  const checkAllRatings = () => {
-    return allRatings.filter(singleRating => {
-      return singleRating.movie_id === movieId
-    })
-  }
-
-return (
-  checkAllRatings().length ?
-  <p>{`Your rating: ${checkAllRatings()[0].rating}`}</p> :
-  <section>
-    <button id="1" onClick={(event) => handleClick(event)}>star1</button>
-    <button id="2" onClick={(event) => handleClick(event)}>star2</button>
-    <button id="3" onClick={(event) => handleClick(event)}>star3</button>
-    <button id="4" onClick={(event) => handleClick(event)}>star4</button>
-    <button id="5" onClick={(event) => handleClick(event)}>star5</button>
-    <button id="6" onClick={(event) => handleClick(event)}>star6</button>
-    <button id="7" onClick={(event) => handleClick(event)}>star7</button>
-    <button id="8" onClick={(event) => handleClick(event)}>star8</button>
-    <button id="9" onClick={(event) => handleClick(event)}>star9</button>
-    <button id="10" onClick={(event) => handleClick(event)}>star10</button>
-  </section>
-  )
+  };
 
+  const getUserRating = movieId => {
+    const userRatings = allRatings.map(rating => rating.movie_id);
+      if (userRatings.includes(movieId)) {
+        return allRatings.find(movie => movie.movie_id === movieId).rating;
+      } else {
+        return '...';
+    }
+  };
+
+  return (
+    <>
+    {getUserRating(movieId) !== '...' &&
```

Comment:
@allisonjw This button works on the MovieCard, but not on the ShowPage.  I think we should still merge this and just create an issue.

<img width="1439" alt="Screen Shot 2020-01-05 at 8 49 50 PM" src="https://user-images.githubusercontent.com/24462031/71794123-522af600-2ffd-11ea-8aee-31c340c74faa.png">